mongoData.js

```diff
@@ -0,0 +1,94 @@
+const { MongoClient } = require('mongodb');
+const connection_string = "mongodb+srv://anujsahu:1MBDz4bok7B5Ixug@cluster0.qomzy21.mongodb.net/?retryWrites=true&w=majority";
```

Comment:
The MongoDB connection string contains a password which is a serious security issue. Remove the password from the code and use environment variables instead. [important]